README.md

```diff
@@ -181,6 +181,13 @@ To add a resource or contribute, please check out our contribution guide [here](
 
 ---
 
+## C++ Algorithms
```

Comment:
Rename to just C++

Reply:
I have updated it. Just Check if any further changes are required